internal/v2/repository/iam/policy/label/read_model.go

```diff
@@ -0,0 +1,21 @@
+package label
+
+import (
+	"github.com/caos/zitadel/internal/eventstore/v2"
+	"github.com/caos/zitadel/internal/v2/repository/policy/label"
+)
+
+type LabelPolicyReadModel struct{ label.LabelPolicyReadModel }
+
+func (rm *LabelPolicyReadModel) AppendEvents(events ...eventstore.EventReader) {
+	for _, event := range events {
+		switch e := event.(type) {
+		case *LabelPolicyAddedEvent:
+			rm.ReadModel.AppendEvents(&e.LabelPolicyAddedEvent)
+		case *LabelPolicyChangedEvent:
+			rm.ReadModel.AppendEvents(&e.LabelPolicyChangedEvent)
+		case *label.LabelPolicyAddedEvent, *label.LabelPolicyChangedEvent:
```

Comment:
this case should not be need. mapper functions should be located in the aggregate package. e.g. mapper for member is located in iam and the abstract function in member